diff -Naur ChezScheme-9.5.4/makefiles/Mf-install.in ChezScheme-9.5.4-Fedora/makefiles/Mf-install.in
--- ChezScheme-9.5.4/makefiles/Mf-install.in	2020-08-31 04:35:12.000000000 +0200
+++ ChezScheme-9.5.4-Fedora/makefiles/Mf-install.in	2020-11-20 15:31:37.750789601 +0100
@@ -107,7 +107,7 @@
 I=./installsh -o "${InstallOwner}" -g "${InstallGroup}"
 
 bininstall: ${Bin}
-	$I -m 555 ${Scheme} ${SchemePath}
+	$I -m 755 ${Scheme} ${SchemePath}
 	ln -f ${SchemePath} ${PetitePath}
 	ln -f ${SchemePath} ${SchemeScriptPath}
 
@@ -123,12 +123,12 @@
           ln -f ${LibBin}/scheme.boot ${LibBin}/${InstallSchemeName}.boot;\
         fi
 	ln -f ${LibBin}/scheme.boot ${LibBin}/${InstallScriptName}.boot;
-	$I -m 444 ${Include}/main.o ${LibBin}
+	$I -m 644 ${Include}/main.o ${LibBin}
 	$I -m 444 ${Include}/scheme.h ${LibBin}
 	$I -m 444 ${Revision} ${LibBin}/revision
 
 installkernelobj: ${LibBin}
-	$I -m 444 ${Include}/kernel.o ${LibBin}
+	$I -m 644 ${Include}/kernel.o ${LibBin}
 
 installkernellib: ${LibBin} ${InstallZlibTarget} ${InstallLZ4Target}
 	$I -m 444 ${Include}/libkernel.a ${LibBin}
